src/main/java/core/basesyntax/FruitStorage.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitStorage {
+    public Map<String, Integer> fruitStorage(List<Operation> operations) {
+        Map<String, Integer> fruitStorage = new HashMap<>();
+        Map<String, LocalDate> expiration = new HashMap<>();
```

Comment:
i suggest we declare these variables in the class, not method, so they can be reused + in order to stick with SRP this class should only be responsible for storing fruits, not logic with operation, let's split this code

Reply:
splitted